### PR TITLE
include ability to create queries with Loan Tags from an LSC. Minor t…

### PR DESCRIPTION
**Pull request by mcstover**
…weaks and clean up to campaign page and components

The update to include loanTags should start fixing up ML2-108

### CODE REVIEW

File: src/components/CorporateCampaign/CampaignLoanDisplay.vue [hunk 1]
```diff
@@ -11,14 +11,15 @@
 		</div> -->
 		<div class="columns small-12 large-8 align-self-middle" v-if="showLoans && loans.length > 0">
 			<div class="loan-card-group row small-up-1 large-up-2 xxlarge-up-3">
+				<!-- LendHomepageLoanCard -->
```
Comment (mcstover) [inline src/components/CorporateCampaign/CampaignLoanDisplay.vue:14]: temporary switch due to removal of action button

File: src/api/fixtures/LoanSearchFilters.js [hunk 2]
```diff
@@ -68,10 +69,12 @@ export function getInputFilters(filters) {
 // Return filters that can be used in a loan search
 export function getSearchableFilters(filters) {
```
Comment (mcstover) [inline src/api/fixtures/LoanSearchFilters.js:70]: This is the only method I'm currently using to create a loanQuery for the campaign pages.

Comment (mcstover) [reply src/api/fixtures/LoanSearchFilters.js:70]: Addition of `loanTags` in the other methods don't appear to have any affect on the functionality of auto lending so I think they are ok.

**Review by emuvente — APPROVED** [review]
👍  it's cool to see the fixtures being expanded upon!

**Pull request merged**